cmd/aether-mock-exporter/aether-mock-exporter.go

```diff
@@ -0,0 +1,98 @@
+// SPDX-FileCopyrightText: 2020-present Open Networking Foundation <info@opennetworking.org>
```

Comment:
probably should be Intel attribution, since this is new file.
Same for others

Reply:
Goes to the definition of "new". It's a new filename, but the contents themselves are old. Nevertheless, I think adding the Intel header (while keeping the ONF one) is probably a good idea. Let me dig up your notes in Teams.